Baby Monitor/Source Files/Modules/Server/ServerViewController.swift

```diff
@@ -117,4 +133,10 @@ final class ServerViewController: BaseViewController {
         localVideoTrack = stream.videoTracks[0] as? RTCVideoTrack
         localVideoTrack?.add(localView)
     }
+
+    private func fireVideoTimer() {
+        videoTimer = Observable<Int>.interval(Constants.videoStreamingHiddenAfterTime,
+                                              scheduler: MainScheduler.instance)
+            .startWith(0)
```

Comment:
Is `.startWith(0)` necessary? It seems that you're skipping it anyway on subscription

Reply:
Nice catch!